CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [10.1.0] IN PROGRESS
 
 * Also support `feesfines` interface version `19.0`. Refs UIU-2960.
+* Can not edit shadow user account. Refs UIU-3000.
```

Comment:
Could you rephrase this description to indicate what has been implemented rather than the problem statement?